src/dmd/cli.d

```diff
@@ -754,4 +758,47 @@ CPU architectures supported by -mcpu=id:
         }();
         return s;
     }
+
+    /// Options supported by -checkaction=
+    static string checkActionUsage()
+    {
+        return "
+Behavior on assert/boundscheck/finalswitch failure:
+  =?             List information on all available choices
+  =D             Usual D behavior of throwing an Error
+  =C             Call the C runtime library assert failure function
+  =halt          Halt the program execution (very lightweight)
+  =context       Use D assert with context information (when available)
+";
+    }
+
+    /// Options supported by -check
+    static string checkUsage()
+    {
+        return "
+Enable or disable specific checks:
+  =?                    List information on all available choices
+  =assert[=[on|off]]    Assertion checking
+  =bounds[=[on|off]]    Array bounds checking
+  =in[=[on|off]]        In contracts
+  =invariant[=[on|off]] Class/struct invariants
+  =out[=[on|off]]       Out contracts
+  =switch[=[on|off]]    Switch defaults
+  =on                   Enable all assertion checking
+                        (default for non-release builds)
+  =off                  Disable all assertion checking
+";
+    }
+
+    static string stdCppUsage()
+    {
+        return "
+Available C++ standards:
+  =?                    List information on all available choices
+  =c++98                Sets `__traits(getTargetInfo, \"cppStd\")` to `199711`
+  =c++11                Sets `__traits(getTargetInfo, \"cppStd\")` to `201103`
+  =c++14                Sets `__traits(getTargetInfo, \"cppStd\")` to `201402`
+  =c++17                Sets `__traits(getTargetInfo, \"cppStd\")` to `201703`
+";
+    }
```

Comment:
Because I want to generate these pages too (as it's e.g. done for `-transition`) to ensure that the HTML, CLI and MAN page are in sync, but that was too much for this PR (it already got quite large).
Also, for example, for `-transition` the available options are currently not listed in the MAN or HTML page, so that's going to be a follow-up anyhow.

Lastly, the the overhead of calling a function doesn't matter if you print stuff to stdout and then exit the program (though probably LDC is smart enough to optimize this away anyhow).

Reply:
Ok.

Reply:
I think a spade should be called a spade. This is data, if you need to dress it up quickly in a function just put a lambda around it.

Reply:
Well these functions won't exist for long as I plan to generate their text anyhow soon when this is in... but I changed it anyhow.